examples/13_two_tensor_op_fusion/device/b2b_gemm.h

```diff
@@ -233,12 +240,18 @@ class B2bGemm {
       TensorRef<ElementB const, LayoutB> ref_B1_,
       TensorRef<ElementC const, LayoutC> ref_C1_,
       TensorRef<ElementC, LayoutC> ref_D1_,
-      typename EpilogueOutputOp0::Params epilogue0_ = 
+      int64_t batch_stride_A0 = 1,
+      int64_t batch_stride_B0 = 1,
+      int64_t batch_stride_B1 = 1,
+      int64_t batch_stride_C1 = 1,
+      int64_t batch_stride_D1 = 1,
+      typename EpilogueOutputOp0::Params epilogue0_ =
         typename EpilogueOutputOp0::Params(),
-      typename EpilogueOutputOp1::Params epilogue1_ = 
+      typename EpilogueOutputOp1::Params epilogue1_ =
         typename EpilogueOutputOp1::Params(),
       int split_k_slices_ = 1
```

Comment:
Please add a check to make sure that gemm_batched and split_k_slices are not used at the same time.